voucher/src/main/java/org/devcourse/voucher/Main.java

```diff
@@ -0,0 +1,8 @@
+package org.devcourse.voucher;
+
+public class Main {
+    public static void main(String[] args) {
+        CommandLineApplication commandLineApplication = new CommandLineApplication();
+        commandLineApplication.run();
+    }
+}
```

Comment:
```suggestion
@Component
public class Main implements ApplicationRunner {
    @Override
    public void run(ApplicationArguments args) throws Exception {
        Console console = new Console();
        console.printMenu();
        String option = console.getOption();
        System.out.println(option);
    }
}
```
`ApplicationRunner`를 활용하면 깔끔해질 것 같네요. 그리고 Main 클래스도 Console 애플리케이션과 관련된 것이라 패키지 이동이 필요해보입니다. 클래스 이름도 명확하게 지어주세요
- https://www.daleseo.com/spring-boot-runners/